packages/ui/avatar/avatarSystem.ts

```diff
@@ -76,6 +76,7 @@ export class AvatarEntity extends Entity {
       // this should be deleted once this PR is merged and implemented in renderer
       // https://github.com/decentraland/protocol/pull/39
       if (profile.avatar.emotes) {
+        shape.emotes = profile.avatar.emotes.map(($) => $.urn)
```

Comment:
is it okay that the slots are not being sent?

Reply:
slots are part of the local profile, the SDK components should be as simple as possible